lib/service/servicecfg/app.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package servicecfg
```

Comment:
Wouldn't just `cfg` be a more appropriate package name or did you name it this way to avoid using import aliases?

Reply:
I named it this way because `cfg` is very vague.

There's existing precedent:

- `io/ioutil`
- `net/http/httptest`
- `net/http/httptrace`